docs/en/pact-functions.md

```diff
@@ -1353,6 +1353,22 @@ Commit transaction.
 ```
 
 
+### continue-pact {#continue-pact}
+
+*pact-id*&nbsp;`integer` *step*&nbsp;`integer` *&rarr;*&nbsp;`string`
+
+*pact-id*&nbsp;`integer` *step*&nbsp;`integer` *rollback*&nbsp;`bool` *&rarr;*&nbsp;`string`
+
+*pact-id*&nbsp;`integer` *step*&nbsp;`integer` *rollback*&nbsp;`bool` *yielded*&nbsp;`object:[]<{y}>` *&rarr;*&nbsp;`string`
+
+
+Continue previously-initiated pact identified by PACT-ID at STEP, optionally specifying ROLLBACK (default is false), and YIELDED value to be read with 'resume' (if not specified, uses yield in most recent pact exec, if any).
+```lisp
+(continue-pact 2 1 true)
+(continue-pact 2 1 false { "rate": 0.9 })
+```
+
+
```

Comment:
So the yielded can be either `object` or `string` or unspecified ya? Can we get an example of not supplying either a rollback or yielded value? I imagine most usage will be `(continue-pact 2 1)`.

Reply:
No, `yield` only allows an object. Added more examples.